gtsfm/evaluation/metrics_report.py

```diff
@@ -0,0 +1,198 @@
+import os
+from typing import Any, Dict, List, Tuple, Union
+
+import plotly.graph_objects as go
```

Comment:
module level docstring would be nice. What does this file do, in general? why do we need graph objects and tabulate?

Reply:
added, these are functions that do the plotting and tabulating.